easybuild/easyconfigs/i/iompi/iompi-2016.09-GCC-4.9.3-2.25.eb

```diff
@@ -11,9 +11,9 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
 compver = '2016.3.210-GCC-4.9.3-2.25'
 
 dependencies = [
-    ('OpenMPI', '1.10.4', '', ('iccifort', compver)),
     ('icc', compver),
     ('ifort', compver),
+    ('OpenMPI', '1.10.4', '', ('iccifort', compver)),
```

Comment:
I'm OK with this change, but I'm wondering what made you do this? Just consistency with existing easyconfig files?